### PR TITLE
[docs] Use Netflix financial results to document bar charts

### PR DESCRIPTION
**Pull request by alexfauquette**
I took Netflix financial results from Yahoo Finance and used it to illustrate bar chart staking. The underlying idea is that financial data are composed of sub-data. For example, assets are divided into short-term and long-term ones, which is well suited for staked bars.

But I'm uncertain this helps to make the docs more appealing and do not disturb the learning aspect

image

https://deploy-preview-13991--material-ui-x.netlify.app/x/react-charts/bars/#bar-size
https://deploy-preview-13991--material-ui-x.netlify.app/x/react-charts/bars/#stacking

### CODE REVIEW

File: docs/data/charts/bars/StackBars.js [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+        { dataKey: 'currAss', stack: 'assets' },
+        { dataKey: 'nCurrAss', stack: 'assets' },
+        { dataKey: 'curLia', stack: 'liability' },
+        { dataKey: 'nCurLia', stack: 'liability' },
+        { dataKey: 'capStock', stack: 'equity' },
+        { dataKey: 'retEarn', stack: 'equity' },
+        { dataKey: 'treas', stack: 'equity' },
+      ])}
+      xAxis={[{ scaleType: 'band', dataKey: 'year' }]}
+      slotProps={{ legend: { hidden: true } }}
```
Comment (JCQuintas) [inline docs/data/charts/bars/StackBars.js:19]: Do we need to remove it? Seems like it could be helpful for the user to understand how the charts work 🤔

**Review by JCQuintas — APPROVED** [review]
Small comment, otherwise looks good

File: docs/data/charts/bars/StackBars.js (hunk 1 above)
Comment (alexfauquette) [reply docs/data/charts/bars/StackBars.js:19]: The legend started to take too much space compared to the feature to display

image

File: docs/data/charts/bars/netflixsBalanceSheet.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
+const translations = {
```
Comment (alexfauquette) [inline docs/data/charts/bars/netflixsBalanceSheet.ts:1]:
```suggestion
// Data from https://finance.yahoo.com/quote/NFLX/financials/

const translations = {
```

**Pull request merged**